lang/csharp/versions.props

```diff
@@ -27,14 +27,16 @@
   -->
   <PropertyGroup Label="Latest Package Versions">
     <NewtonsoftJsonVersion>13.0.1</NewtonsoftJsonVersion>
-    <SystemCodeDomVersion>7.0.0</SystemCodeDomVersion>
+    <SystemCodeDomVersion>8.0.0</SystemCodeDomVersion>
     <SystemReflectionVersion>4.3.0</SystemReflectionVersion>
     <SystemReflectionEmitILGenerationVersion>4.7.0</SystemReflectionEmitILGenerationVersion>
     <SystemReflectionEmitLightweightVersion>4.7.0</SystemReflectionEmitLightweightVersion>
 
     <!-- The following packages are required for the extra codec libraries. These are not direct dependencies of the Avro.main library. -->
-    <SharpZipLibVersion>1.4.1</SharpZipLibVersion>
-    <IronSnappyVersion>1.3.0</IronSnappyVersion>
+    <SharpZipLibVersion>1.4.2</SharpZipLibVersion>
+    <IronSnappyVersion>1.3.1</IronSnappyVersion>
+    <!-- As of 11/19/2023, Joveler.Compression.XZ 4.3+ is not compatible with the Linux ARM64 C# github workflow.
+    Keep it at 4.1.0 until the issue is resolved. Maybe the shipped liblzma.so is incompatible? -->
```

Comment:
Is it worth it to report the issue to the Joveler developers ?

Reply:
Yes, I will. I just wanted to reproduce and investigate before I do that.